modules/installation-aws-permissions.adoc

```diff
@@ -264,19 +269,19 @@ If you use an existing VPC, your account does not require these permissions to d
 .Additional IAM and S3 permissions that are required to create manifests
 [%collapsible]
 ====
-* `iam:DeleteAccessKey`
-* `iam:DeleteUser`
-* `iam:DeleteUserPolicy`
-* `iam:GetUserPolicy`
-* `iam:ListAccessKeys`
-* `iam:PutUserPolicy`
-* `iam:TagUser`
-* `s3:PutBucketPublicAccessBlock`
-* `s3:GetBucketPublicAccessBlock`
-* `s3:PutLifecycleConfiguration`
-* `s3:HeadBucket`
-* `s3:ListBucketMultipartUploads`
-* `s3:AbortMultipartUpload`
+* `iam:DeleteAccessKey` # Also listed in the Required permissions to delete base cluster resources table, which matches the go file.
```

Comment:
I guess it's fine to repeat some perms in different sections because customers can use different roles for install vs destroy, for example.
Not sure why some `iam:Delete*` perms are need for manifests creation, though.